frontend/src/old-pages/Configure/Source.tsx

```diff
@@ -130,7 +130,11 @@ function sourceValidate(suppressUpload = false) {
 
 function ClusterSelect() {
   const selectedPath = ['app', 'wizard', 'source', 'selectedCluster']
+  const apiVersion = useState(['app', 'version', 'full'])
   const clusters = useState(['clusters', 'list']) || []
+  const filteredClusters = clusters.filter(
+    (cluster: ClusterInfoSummary) => cluster.version === apiVersion,
```

Comment:
question: are we sure we want to do an exact version check here? Only minor versions of PC introduce new features and fields, so we can be probably safe allowing an import between patch versions.
For example, if I'm running PC 3.2.1 and I want to import a cluster to from 3.2.0, PCM should allow that.

Reply:
Agree, we should be safe. Going to check up to the minor only.

Reply:
I'm not sure about this for the time being, because whilst it's true that a patch version should contain only bug fixes we still don't have a clear definition of what a patch version might be. For the time being I would keep it safe and maybe widen the scope with a subsequent release, when we will have an agreement of what a patch version is.

Reply:
Shouldn't be intrinsic what a patch version is in semver? Or PC is not adhering completely to the specific?

Reply:
SemVer is followed correctly if you take a look at the [release history](https://docs.aws.amazon.com/parallelcluster/latest/ug/document_history.html)